silkworm/infra/common/timer_test.cpp

```diff
@@ -25,14 +25,14 @@
 namespace silkworm {
 
 struct TimerTest {
-    const std::vector<uint32_t> kIntervals{100, 10, 1};  // milliseconds
+    const std::vector<uint32_t> intervals{100, 10, 1};  // milliseconds
```

Comment:
this is meant to be a constant, maybe add constexpr or static?

Reply:
done